conf/nginx.conf.tpl

```diff
@@ -39,6 +44,12 @@ http {
     rewrite ^/minutes/(\d+)/(.*) https://hypothesis-meeting-logs.s3.amazonaws.com/$1/$2 redirect;
     rewrite ^/minutes/(.*) https://shrub.appspot.com/hypothesis-meeting-logs/$1 redirect;
 
+
+    # Set the fallthrough upstream URL.
+    # We store the upstream endpoint in a variable to trigger dynamic resolution
+    # of the upstream hostname.
+    set $fallthrough_upstream ${FALLBACK_UPSTREAM};
```

Comment:
Just to save my poor brain, can we refer to this as `fallback_upstream` for consistency?